Makefile

```diff
@@ -64,7 +64,6 @@ TARGETS += drbd
 TARGETS += ecr-credential-provider
 TARGETS += fuse3
 TARGETS += gasket-driver
-TARGETS += crun
```

Comment:
hmm weird, probably forgot to add in `.kres.yaml`

Reply:
Added to `.kres.yaml`